src/main/scala/ru/smslv/akka/dns/raw/DnsClient.scala

```diff
@@ -45,6 +47,16 @@ class DnsClient(ns: InetSocketAddress, upstream: ActorRef) extends Actor with Ac
         log.debug(s"Message to $ns: $msg6")
         socket ! Udp.Send(msg6.write(), ns)
 
+      case SrvQuestion(id, name) =>
```

Comment:
Would you consider renaming to `SRVQuestion` to match other SRV related class names?